VERSION_MANAGERS.md

```diff
@@ -1,8 +1,32 @@
-# Custom version managers
+## Chruby
```

Comment:
NIT: let's keep custom activation on top.

Reply:
IMO notes about officially supported managers should appear before custom activations. It also matches how we phrase it in the readme: 

> For manager-specific notes and how to set up custom activation for managers that are not listed above